modules/wallet/transactionbuilder.go

```diff
@@ -660,6 +662,23 @@ func (w *Wallet) SpendableOutputs() []modules.SpendableOutput {
 		})
 	})
 
+	// don't include outputs marked as spent in pending transactions
```

Comment:
@DavidVorick this seemed like a reasonable thing to do, but would like to hear your thoughts